pubspec.lock

```diff
@@ -1445,8 +1445,8 @@ packages:
     dependency: "direct main"
     description:
       path: "."
-      ref: improve-matrix-file
-      resolved-ref: b8762469be46d3ec6ee782ccdc035a597202c2f4
+      ref: "hot-fix/failed-to-build-web"
```

Comment:
you must add the PR of dart sdk to the desc. So you can update this lock after the dependent PR merged

Reply:
What do you mean ? @@